### ISSUE TITLE
[BUG] Fails to load in Home Assistant 2022.7.0b0

### ISSUE THREAD

**Issue by firstof9** [travisghansen/hass-pfsense#81]
Log:
```
2022-06-29 13:27:50 ERROR (MainThread) [homeassistant.components.sensor] Error adding entities for domain sensor with platform pfsense
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 397, in async_add_entities
    await asyncio.gather(*tasks)
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 641, in _async_add_entity
    await entity.add_to_platform_finish()
  File "/usr/src/homeassistant/homeassistant/helpers/entity.py", line 775, in add_to_platform_finish
    await self.async_added_to_hass()
  File "/usr/src/homeassistant/homeassistant/helpers/update_coordinator.py", line 332, in async_added_to_hass
    self._handle_coordinator_update, self.coordinator_context
AttributeError: 'PfSenseStaticKeySensor' object has no attribute 'coordinator_context'
```

**Comment by aAbdman** [comment]
Same problem here

**Comment by travisghansen** [comment]
I’ll take a peek. Looking to do some maintenance on this project over the next few days.

I briefly fired up the latest beta however and did not see this issue.

**Comment by travisghansen** [comment]
should be fixed in `v0.5.0` with this commit: https://github.com/travisghansen/hass-pfsense/commit/c557539b2cbce63d91df805bd7e12cdbc47afa23
